### PR TITLE
Add checks for polkadot and/or polkadot-parachain `not found` errors

### LINKED ISSUE #388: Beautify the `polkadot: command not found` error
Make the error below, to provide a "proposing" solution of running `setup` and/or make sure that the polkadot command is correctly added in the PATH.

```bash
Error: Command failed with exit code 127: /bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json
/bin/bash: line 1: polkadot: command not found
    at makeError (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  shortMessage: 'Command failed with exit code 127: /bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json',
  command: '/bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json',
  escapedCommand: '"/bin/bash" -c "polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json"',
  exitCode: 127,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '/bin/bash: line 1: polkadot: command not found',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
Error: Command failed with exit code 127: /bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json
/bin/bash: line 1: polkadot: command not found
    at makeError (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  shortMessage: 'Command failed with exit code 127: /bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json',
  command: '/bin/bash -c polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json',
  escapedCommand: '"/bin/bash" -c "polkadot build-spec --chain rococo-local --disable-default-bootnode > /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/cfg/rococo-local-plain.json"',
  exitCode: 127,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '/bin/bash: line 1: polkadot: command not found',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}

	 Node's logs are available in /tmp/zombie-015eaebaf59ef306c6f0250230dda65d_-322338-qPwpUzehoTAB/logs
Error: Command failed with exit code 2: /bin/bash -c kill -9 
kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]
    at makeError (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/home/wirednkod/Documents/repos/parity/zombienet/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {
  shortMessage: 'Command failed with exit code 2: /bin/bash -c kill -9 ',
  command: '/bin/bash -c kill -9 ',
  escapedCommand: '"/bin/bash" -c "kill -9 "',
  exitCode: 2,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'kill: usage: kill [-s sigspec | -n signum | -sigspec] pid | jobspec ... or kill -l [sigspec]',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

### PR DESCRIPTION
**Pull request by wirednkod**
Fixes #388. See screenshot below on respective message:

image

Note: This is a nice start on "grouping" the ERROR messages into Utils package

### CODE REVIEW

File: javascript/packages/orchestrator/src/configGenerator.ts [hunk 1]
```diff
@@ -240,7 +239,7 @@ export async function generateNetworkSpec(
       const isCumulusBased =
         parachain.cumulus_based !== undefined
           ? parachain.cumulus_based
-          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
+          : ![DEFAULT_COLLATOR_IMAGE, UNDYING_COLLATOR_BIN].includes(
```
Comment (pepoviola) [inline javascript/packages/orchestrator/src/configGenerator.ts:242]:
```suggestion
          : ![DEFAULT_ADDER_COLLATOR_BIN, UNDYING_COLLATOR_BIN].includes(
```
We should check agains the adder-collator command.

Comment (wirednkod) [reply javascript/packages/orchestrator/src/configGenerator.ts:242]: Ah sorry. This was erroneously altered.. Thank you!

File: javascript/packages/orchestrator/src/configGenerator.ts [hunk 2]
```diff
@@ -9,7 +9,6 @@ import {
 } from "@zombienet/utils";
 import {
   ARGS_TO_REMOVE,
-  DEFAULT_ADDER_COLLATOR_BIN,
```
Comment (pepoviola) [inline javascript/packages/orchestrator/src/configGenerator.ts:12]: I think you need to import `DEFAULT_ADDER_COLLATOR_BIN` to use it in the check.

Comment (wirednkod) [reply javascript/packages/orchestrator/src/configGenerator.ts]: That was a erroneous merge that should not have happened. Thank you for noticing!

**Review by pepoviola — APPROVED** [review]

**Pull request merged**